gnmi_server/server.go

```diff
@@ -274,26 +284,32 @@ func (s *Server) checkEncodingAndModel(encoding gnmipb.Encoding, models []*gnmip
 
 // Get implements the Get RPC in gNMI spec.
 func (s *Server) Get(ctx context.Context, req *gnmipb.GetRequest) (*gnmipb.GetResponse, error) {
+	common_utils.IncCounter("GNMI get")
```

Comment:
>IncCounter [](http://example.com/codeflow?start=14&length=10)

The counter and dump are quite useful new features, not directly tie to the native mode gnmi service feature. Could you separate them into separate PR(s)?